contracts/gauges/LiquidityGaugeV5Collateral.vy

```diff
@@ -496,10 +501,12 @@ def withdraw(_value: uint256, _claim_rewards: bool = False):
 
         total_supply -= _value
         new_balance: uint256 = self.balanceOf[msg.sender] - _value
+        new_adjusted_balance: uint256 = self.adjustedBalanceOf[msg.sender] - _value
         self.balanceOf[msg.sender] = new_balance
+        self.adjustedBalanceOf[msg.sender] = new_adjusted_balance
         self.totalSupply = total_supply
 
-        self._update_liquidity_limit(msg.sender, new_balance, total_supply)
+        self._update_liquidity_limit(msg.sender, new_adjusted_balance, total_supply)
```

Comment:
Why are we not updating `balanceOf`?  If Alice withdraws all her tokens, `balanceOf` and `adjustedBalanceOf` would both go to zero?